DocumentationToolingNotes.md

```diff
@@ -0,0 +1,62 @@
+# How the Office JavaScript API documentation is generated
+
+The Office JavaScript reference documentation pages are generated from type definition files and example snippets. This process uses a blend of open source tools and repository-specific scripts. This document aims to make the processes of this repository transparent so the community can better benefit from and contribute to this content.
+
+## Content sources
+
+Two types of content are combined to create the Office-JS reference documentation: type definitions and code snippets. These ensure complete API coverage and give small, inline code samples.
+
+### Type definition files
+
+The type definition files on [Definitely Typed](https://github.com/DefinitelyTyped/DefinitelyTyped) serve as the source of truth our documentation. Any Office Add-in that uses TypeScript compiles against one or more of these type definition files. These also give JavaScript and TypeScript developers IntelliSense capabilities. By powering the reference documentation with these definitions, we limit the possibility of incorrect information.
+
+There are four relevant d.ts files powering different subsections of the docs.
+
+- [office-js/index.d.ts](https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js/index.d.ts) (The Release definitions.)
+  - [Excel (Release)](https://docs.microsoft.com/javascript/api/excel_release)
+  - [OneNote](https://docs.microsoft.com/javascript/api/onenote)
+  - [PowerPoint](https://docs.microsoft.com/javascript/api/visio)
+  - [Visio](https://docs.microsoft.com/javascript/api/visio)
+  - [Word (Release)](https://docs.microsoft.com/javascript/api/word_release)
+  - [OfficeExtensions subsection of the Common API](https://docs.microsoft.com/javascript/api/office)
+- [office-js-preview/index.d.ts](https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js-preview/index.d.ts) (The Preview definitions.)
+  - [Excel (Preview)](https://docs.microsoft.com/javascript/api/excel)
```

Comment:
None of these links look like preview docs, so not sure if these are correct or not.

Reply:
The preview docs include all the previous APIs too.